src/Evolution/Particles/MonteCarlo/TakeTimeStep.tpp

```diff
@@ -0,0 +1,125 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Evolution/Particles/MonteCarlo/CellVolume.hpp"
+#include "Evolution/Particles/MonteCarlo/TemplatedLocalFunctions.hpp"
```

Comment:
make this the first include, with a blank line after it. 

Also add the appropriate includes